NGitLab/Models/CompareQuery.cs

```diff
@@ -15,6 +17,22 @@ public class CompareQuery
     /// </summary>
     public string Target { get; set; }
 
+    /// <summary>
+    /// Comparison method: true for direct comparison between from and to (from..to), false to compare using merge base (from…to)’. Default is false.
+    /// </summary>
+    public bool Straight { get; set; }
```

Comment:
Since [`straight` is an optional query parameter](https://docs.gitlab.com/api/repositories/#compare-branches-tags-or-commits) (and even though its default implicit value is `false`), I would make it nullable. Same thing for `unidiff`.